modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -157,12 +157,15 @@ To review a package EULA, open the package detail page in the {webui}.
 Moved to System Locking chapter --LKB 2020-07-17
 // Why do we talk about Package Locks here? ke, 2019-12-18
 // see https://github.com/uyuni-project/uyuni-docs/issues/41
+// 2021-08-26, ke: do we want to uncomment this now?
 == Package Locks
 
 [NOTE]
 ====
-Package locks can only be used on traditional clients that use the Zypper package manager.
-The feature is not currently supported on {rhel} or Salt clients.
+Package locks can be used on traditional and Salt clients that use the Zypper package manager.
+Package locks can also be used on some other Salt clients.
+On {rhel}-like systems (version 7), you must have the [package]``yum-plugin-versionlock`` package installed.
```

Comment:
On RHEL 8 locking is supported as well. The plugin package is named a bit different:

  python3-dnf-plugin-versionlock

Reply:
And Debian/Ubuntu is supported as well. `apt` has the support included.